src/main/java/Duke/Task.java

```diff
@@ -0,0 +1,46 @@
+package Duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    private static char NOT_DONE_SYMBOL = '\u2718';
```

Comment:
this was previously defined in `Duke`, you may want to consider sharing it between both

Maybe create a `Symbol` class just for this case

Reply:
Okay! Will implement it in the next commit!